.zuul.yaml

```diff
@@ -106,3 +121,7 @@
             branches: release-1.11
         - cloud-provider-openstack-acceptance-test-e2e-conformance-stable-branch-v1.12:
             branches: release-1.12
+        - cloud-provider-openstack-acceptance-test-e2e-conformance-stable-branch-v1.13:
```

Comment:
Please remove 1.10, 1.11 job refers at here, these releases have been [end of life](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/#support-timeframes), only enable latest three release 1.14, 1.13, 1.12 and master jobs to save resource for OpenLab too.

Reply:
ack